src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,26 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+  margin-bottom: 0;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &--1 .stars__star:nth-child(-n+1),
+  &--2 .stars__star:nth-child(-n+2),
+  &--3 .stars__star:nth-child(-n+3),
+  &--4 .stars__star:nth-child(-n+4),
+  &--5 .stars__star:nth-child(-n+5) {
```

Comment:
Try to use [for cycle](https://sass-lang.com/documentation/at-rules/control/for)